### PR TITLE
Reflect skipValidateRequest when http-openapi3 option is enabled

### PR DESCRIPTION
**Pull request by k2tzumi**
refs https://zenn.dev/katzumi/scraps/f00a2d3e177b77#comment-36b36b5507b10d

### CODE REVIEW

**Comment by k2tzumi** [comment]
I tested it on a project with runn installed and it seemed to work fine.

File: operator.go [hunk 1]
```diff
@@ -469,9 +469,17 @@ func New(opts ...Option) (*operator, error) {
 				if key != "" && key != k {
 					continue
 				}
-				val, err := newHttpValidator(&httpRunnerConfig{
+				runner, ok := bk.runners[k].(map[string]interface{})
```
Comment (k2tzumi) [inline operator.go:472]: I don't think it will be any other type, but what do you think?

File: operator.go [hunk 2]
```diff
@@ -469,9 +469,17 @@ func New(opts ...Option) (*operator, error) {
...
+				if !ok {
+					return nil, fmt.Errorf("invalid type: %v", bk.runners[k])
+				}
+				c := &httpRunnerConfig{
 					OpenApi3DocLocation: p,
-				})
+				}
+				c.SkipValidateRequest, _ = runner["skipValidateRequest"].(bool)
+				c.SkipValidateResponse, _ = runner["skipValidateResponse"].(bool)
```
Comment (k2tzumi) [inline operator.go:480]: If it's not on yaml, it's assumed to be false.

**Review by k2tzumi — COMMENTED** [review]
@k1LoW 
Is there anywhere else the parameters should be transferred?

File: operator.go (hunk 1 above)
Comment (k1LoW) [inline operator.go:472]: gostyle.ifacenames may be a false positive.

```suggestion
				runner, ok := bk.runners[k].(map[string]any)
```

File: operator.go (hunk 2 above)
Comment (k1LoW) [reply operator.go:480]: 👍

File: operator_test.go [hunk 3]
```diff
@@ -1276,3 +1276,35 @@ func newRunNResult(t *testing.T, total int64, results []*RunResult) *runNResult
...
+	}{
+		{"false", true},
+		{"true", false},
+	}
+	for _, tt := range tests {
+		t.Run(fmt.Sprintf("skipValidateRequest: %s", tt.skipValidateRequest), func(t *testing.T) {
+			ctx := context.Background()
+			ts := testutil.HTTPServer(t)
+			t.Setenv("TEST_HTTP_END_POINT", ts.URL)
+			t.Setenv("SKIP", tt.skipValidateRequest)
```
Comment (k1LoW) [inline operator_test.go:1293]:
```suggestion
			t.Setenv("TEST_SKIP_VALIDATE_REQUEST", tt.skipValidateRequest)
```

It is easy to see from the environment variable names which fields are reflected.

**Review by k1LoW — COMMENTED** [review]
Thank you!!

I'm suggesting a very small correction.

**Review by k1LoW — APPROVED** [review]
LGTM!

**Pull request merged**